AppCreationScripts/sample.json

```diff
@@ -1,54 +1,71 @@
 {
   "Sample": {
```

Comment:
Sam, work with Dogan to see why the Codegenerator wont work for this sample.json file

Reply:
Sure thing. The generator doesn't work properly on this file.

I just committed a working app creation and cleanup script that I hacked together from the output of the generator though. It seems to at least make the app creation script files, but I had to modify them to remove some extra spaces.

Reply:
So the only thing it is not properly generating at this point is the apps.json file (which is only required for quickstart projects, according to Dogan). Dogan has also confirmed that the config and cleanup script included here are able to get the project running fully.